cl_dll/hud_scores.cpp

```diff
@@ -46,6 +47,7 @@ int CHudScores::Draw(float time)
 		return 0;
 
 	const size_t rows_to_draw = static_cast<size_t>(cl_scores->value);
+	const int alpha = static_cast<int>(cl_scores_alpha->value);
```

Comment:
Make it a `uint8_t`, this way it automatically gets bounded to the allowed 0-255 interval. Otherwise, LGTM.